app/views/customers/show.html.erb

```diff
@@ -6,5 +6,27 @@
   <%= link_to "Edit this customer", edit_customer_path(@customer) %> |
   <%= link_to "Back to customers", customers_path %>
 
-  <%= button_to "Destroy this customer", @customer, method: :delete %>
+  <%= button_to "Delete this customer", @customer, method: :delete %>
+
+    <div id= "orders">
+    <% if @customer.orders.present? %>
+        <h1> Customer's Personal Orders </h1>
+
+        <% @customer.orders.each do |order| %>
+          <%= render order%>
+          <p>
+          <%= link_to "Show Order", order %>
+          </p>
+
+          <p>
+          <%= link_to "Edit Order", edit_order_path(order) %>
+          </p>
+
+          <p>
+          <%= link_to "Delete this Order", data: {"turbo-method": :delete} %>
```

Comment:
this isn't working for me but I'm pretty sure that's not on you. it's sending a GET request instead of a DELETE because of some kinks with hotwired or turbo and the `application.js` file that we're working out new directions to. :)  since this is an older assignment if you run into this again let me know and we'll fix it in that app!